run_with_lock.sh

```diff
@@ -9,11 +10,7 @@ if mkdir ${LOCK_DIR} 2>/dev/null; then
   # If the ${LOCK_DIR} doesn't exist, then start working & store the ${PID_FILE}
   echo $$ > ${PID_FILE}
 
-  echo "Changing directory to "
-  echo $HERE
-  cd $HERE
-
- ./backup.py
+  eval $@
 
   rm -rf ${LOCK_DIR}
   exit
```

Comment:
We can delete this - but we should run bb8 with an explicit container name so that it clashes and fails if it's already running.